pkg/specgen/generate/security.go

```diff
@@ -60,8 +60,8 @@ func setLabelOpts(s *specgen.SpecGenerator, runtime *libpod.Runtime, pidConfig s
 func setupApparmor(s *specgen.SpecGenerator, rtc *config.Config, g *generate.Generator) error {
 	hasProfile := len(s.ApparmorProfile) > 0
 	if !apparmor.IsEnabled() {
-		if hasProfile {
-			return errors.Errorf("Apparmor profile %q specified, but Apparmor is not enabled on this system", s.ApparmorProfile)
+		if hasProfile && s.ApparmorProfile != "unconfined" {
+			return errors.Errorf("AppArmor profile %q specified, but it is not enabled on this system", s.ApparmorProfile)
```

Comment:
`it` refers to the profile but not to AppArmor. Changing the error message should be in a separate commit but I suggest keeping it as is.

Reply:
Yes sure, reverted that line.